btrack/dataio.py

```diff
@@ -771,6 +771,13 @@ def lbep(self):
         logger.info(f'Loading LBEP/{self.object_type}')
         return self._hdf['tracks'][self.obj_type]['LBEPR'][:]
 
+    @h5check_property_exists('tracks')
+    def convert_tracks_to_napari(self, **kwargs):
+        """Return the 'data, properties, graph' to load to napari."""
+        data, properties, graph = utils.tracks_to_napari(self.tracks, **kwargs)
+        self.napari_tracks = (data, properties, graph)
```

Comment:
Can't see any real benefit of having this has two steps
```suggestion
        self.napari_tracks = utils.tracks_to_napari(self.tracks, **kwargs)
```